packages/gatsby-cli/src/init-starter.js

```diff
@@ -286,5 +295,6 @@ module.exports = async (starter: string, options: InitOptions = {}) => {
   })
   if (hostedInfo) await clone(hostedInfo, rootPath)
   else await copy(starterPath, rootPath)
+  await endInstruction(rootPath)
```

Comment:
```suggestion
  successMessage(rootPath)
```

Reply:
Sorry my bad, this is the method so I guess no point of making it async. 
Will change it soon

Reply:
No worries, you should be able to commit all the suggestions 🙂